src/components/team/TeamAnalyticsDashboard.tsx

```
@@ -679,6 +693,18 @@
                 <option value="projected">Projected Score</option>
               </select>
+              {sortedPlayers.length > VIRTUALIZE_THRESHOLD && (
+                <button
+                  onClick={() => { setAccessibleMode(v => !v); setPage(1); }}
+                  className={`px-3 py-2 text-sm font-medium rounded-lg border transition-colors ${
+                    accessibleMode
+                      ? 'bg-emerald-50 text-emerald-700 border-emerald-200'
+                      : 'bg-white text-gray-700 border-gray-200 hover:bg-gray-50'
+                  }`}
+                >
+                  Accessible list
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)